llvm/test/MC/ARM/pcrel-thumb-ldrd-relocs.s

```diff
@@ -0,0 +1,6 @@
+@ RUN: not llvm-mc -filetype=obj -o /dev/null %s 2>&1 -triple=thumbv7   | FileCheck %s
+@ RUN: not llvm-mc -filetype=obj -o /dev/null %s 2>&1 -triple=thumbebv7 | FileCheck %s
+
+   ldrd r0, r1, foo
+
+@ CHECK: :[[#@LINE-2]]:4: error: unsupported relocation type
```

Comment:
The CHECK is usually immediately before or after the instruction, so that we can use -1 +1 instead of -N +N

Reply:
As this check has been merged into `pcrel-ldrd-diff-section.s` I found it quite hard to read when it was placed next to the instructions, with all other checks in the same file. I placed it at the of the file next to the other checks.